ci/eval/outpaths.nix

```diff
@@ -59,9 +60,15 @@ let
           __allowFileset = false;
         };
       };
+
+  nixosJobs = import (path + "/nixos/release.nix") {
+    inherit attrNamesOnly;
+    supportedSystems = if systems == null then [ builtins.currentSystem ] else systems;
```

Comment:
Not sure if there is a point to evaluating/building NixOS tests on Darwin – it is the same NixOS systems, just with the test driver running on Darwin, and that configuration is not built by Hydra. I suppose catching eval regressions there has some value, but I’d likely err on the side of saving the system evals.